server/index.js

```diff
@@ -19,6 +19,51 @@ const jsonMiddleWare = express.json();
 app.use(staticMiddleware);
 app.use(jsonMiddleWare);
 
+app.get('/api/accounts/:accountId', (req, res, next) => {
+  const accountId = req.params.accountId;
+  if (!accountId) {
+    throw new ClientError(400, 'username must be provided');
+  }
```

Comment:
`req.params.accountId` is guaranteed to be truthy, because otherwise this pattern of `/api/accounts/:accountId` would not be matched. But you may want to verify if this is a reasonable account id